tests/test_cmd_list.py

```diff
@@ -30,8 +30,8 @@ def test_integration(tmp_path):
     assert result.stdout == "a\nb\n"
 
 
-def test_found(tmp_path):
-    addons = {"a": {}, "b": {}, "c": {"installable": False}}
+def test_integration_found_exclude_core(tmp_path):
+    addons = {"a": {}, "b": {}, "c": {"installable": False}, "web": {}}
```

Comment:
I suggest keeping the `test_found` test and do a separate test for `--exclude-core-addons` that does not depend on `--select-found`.

Reply:
Don't you think it's better to test both features integrate well? I actually added both to be able to work together.